cmd/build.go

```diff
@@ -39,6 +39,11 @@ var buildCmd = &cobra.Command{
 		if !viper.IsSet("build.prefix") {
 			viper.Set("build.prefix", ".")
 		}
+		if !viper.IsSet("build.binaries") {
+			binaries := [1]map[string]string{}
+			binaries[0] = map[string]string{"name": info.Name, "path": "."}
```

Comment:
Why not just `binaries := []map[string]string{{"name": info.Name, "path": "."}}`?